package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ex3-universitysystem",
+  "version": "1.0.0",
+  "description": "This is my approach to solving exercise 3 of the [Programming Skills](https://github.com/blindma1den/Programming-Skills-Level0/blob/main/level_0.txt) challenge.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Remove this script as we're not using any at the moment.

Reply:
I have removed the test script.